views/layout.html

```diff
@@ -80,21 +94,31 @@
       link.classList.add("text-gray-900", "dark:text-gray-100");
     }
     function updateLogo() {
-      const isDarkMode = window.matchMedia('(prefers-color-scheme: dark)').matches;
-      const logoImages = document.querySelectorAll('#alby-logo');
+      const isDarkMode = window.matchMedia(
+        "(prefers-color-scheme: dark)"
+      ).matches;
+      const logoImages = document.querySelectorAll("#alby-logo");
 
       if (isDarkMode) {
-        logoImages.forEach(logoImage => {
-          logoImage.src = '/public/images/alby-logo-with-text-dark.svg';
+        logoImages.forEach((logoImage) => {
+          logoImage.src = "/public/images/alby-logo-with-text-dark.svg";
         });
       } else {
-        logoImages.forEach(logoImage => {
-          logoImage.src = '/public/images/alby-logo-with-text.svg';
+        logoImages.forEach((logoImage) => {
+          logoImage.src = "/public/images/alby-logo-with-text.svg";
         });
       }
     }
-    window.addEventListener('load', updateLogo);
-    window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', updateLogo);
+    const angleArrow = document.getElementById("angle-arrow");
```

Comment:
I think it would be better UX to be able to click anywhere on the icon or the email address to open the dropdown. Also could "angle-arrow" be renamed maybe to be more specific to the dropdown menu?

Reply:
hi @rolznz , Thanks for feedback. started working on it

Reply:
Hey @rolznz, I have made changes now user can click anywhere on email id to view dropdown.